addons/mail/static/src/js/form_renderer.js

```diff
@@ -2,20 +2,31 @@ odoo.define('mail.form_renderer', function (require) {
 "use strict";
 
 var Chatter = require('mail.Chatter');
+var OwlChatter = require('mail.component.Chatter');
 var FormRenderer = require('web.FormRenderer');
 
 /**
  * Include the FormRenderer to instanciate the chatter area containing (a
  * subset of) the mail widgets (mail_thread, mail_followers and mail_activity).
  */
 FormRenderer.include({
+    dependencies: ['owl'],
+    on_attach_callback: function () {
+        this.chatter_component.__callMounted();
+    },
+    on_detach_callback: function () {
+        this.chatter_component.__callWillUnmount();
+    },
+
     /**
      * @override
      */
     init: function (parent, state, params) {
         this._super.apply(this, arguments);
         this.mailFields = params.mailFields;
         this.chatter = undefined;
+        this.chatter_component = undefined;
+        OwlChatter.env = this.call('owl', 'getEnv');
```

Comment:
could use `OwlMixin.getEnv()`

Reply:
I should try again but I replaced it because it wasn't working